omglol/paste.go

```diff
@@ -0,0 +1,166 @@
+package omglol
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+// Create a Paste object
+func NewPaste(Title, Content string, ModifiedOn int64) *Paste {
+	return &Paste{
+		Title:      Title,
+		Content:    Content,
+		ModifiedOn: ModifiedOn,
+	}
+}
+
+// Returns a string representaion of a Paste
+func (p *Paste) String() string {
+	return fmt.Sprintf("Title: %s, Content: %s, ModifiedOn: %d", p.Title, p.Content, p.ModifiedOn)
+}
+
+// Create a new Paste. See https://api.omg.lol/#token-post-pastes-create-a-new-paste
+func (c *Client) CreatePaste(domain string, paste Paste) error {
+	type pasteRequest struct {
+		Title   string `json:"title"`
+		Content string `json:"content"`
+	}
+
+	p := pasteRequest{
+		Title:   paste.Title,
+		Content: paste.Content,
+	}
+
```

Comment:
Add in `listed` just like in PURLs.
```suggestion
	type pasteRequest struct {
		Title   string `json:"title"`
		Content string `json:"content"`
		Listed  *bool  `json:"listed"`
	}

	p := pasteRequest{
		Title:   paste.Title,
		Content: paste.Content,
	}
	
	if !paste.Listed {
		p.Listed = nil
	} else {
		t := true
		p.Listed = &t
	}

```